### DISCUSSION TITLE
Saving a "Navigation" as a gpx.

### DISCUSSION THREAD

**Discussion by hornetster** [osmandapp/OsmAnd#17443] [General]
Sorry, tried to search, but can't find anything...
I am making "routes" using Navigation, and wondering if it is possible to save as gpx?
I find that making a route using "Plan a Route", is VERY difficult when covering thousands of kilometres??! Or is there a trick to actually searching for a destination, as the only way I can see to use it is drag it out, and make a point on the map... VERY difficult when covering thousands of kilometres. And then how do you put in interim waypoints?
Sorry, I'm probably an idiot....

**Comment by ThomasMeister123** [comment]
I was asking for this long time ago.
When I create a "route" via Navigation then it is NOT possible to save these (maybe 5 Routingpoints) as GPX. OSMAnd is saving the whole navigationTRACK.
I'm looking forward to save a Route (just Start/Destination/Waypoints!) as GPX

**Comment by vshcherb** [comment]
Please check
Intermediate Points - https://docs.osmand.net/docs/user/navigation/setup/route-navigation#intermediate-destinations
Share export GPX - https://docs.osmand.net/docs/user/navigation/setup/route-details#share--export-actions

**Reply by hornetster** [reply]
Thank you! (I think...)
Not real sure why I missed that!

**Reply by ThomasMeister123** [reply]
Not what _I_ looking for. "Share" also exports the whole track and not just the points start/intermediate/destination ;(
